client/deposits/details/test/index.tsx

```diff
@@ -60,6 +61,13 @@ describe( 'Deposit overview', () => {
 				},
 			},
 		};
+		global.wcSettings = {
```

Comment:
I'm curious how changes related to transactions are connected to Deposit-related tests.

Reply:
It appears that the code remains from investigating test failures. I believe we no longer require it.